xonsh/xonfig.py

```diff
@@ -523,6 +527,16 @@ def _info(ns):
             ("encoding errors", env.get("XONSH_ENCODING_ERRORS")),
         ]
     )
+    jup_ksm = jup_kernel = None
+    try:
+        from jupyter_client.kernelspec import KernelSpecManager
+
+        jup_ksm = KernelSpecManager()
+        jup_kernel = jup_ksm.find_kernel_specs()[XONSH_JUPYTER_KERNEL]
+    except (ImportError, KeyError):
```

Comment:
Can you separate this out into a couple of try-except blocks, please, with preferably only one line each? Also, you shouldn't need to use a try-KeyError.  Just use `.get()` or similar

Reply:
using `.get()` means there's only one anticipated exception, so I've simplified.  But if something other than `ImportError` is raised (e.g by `ksm = jupyter_client.kernelspec.KernelSpecManager()`) I'll let `info` simply display ``None``.  The user will get a traceback as soon has s/he tries to actually run the kernel, doesn't need a messy traceback obscuring the info output.